src/ripple/nodestore/ShardSizeTuning.md

```diff
@@ -186,17 +186,27 @@ and nudb.key.
 
 ## Conclusion
 
-We showed that using shards with 8k ledgers leads to raising required disk
-size by 22% in comparison with using shards with 16k ledgers. In the same way,
+We showed that using shards with 8k ledgers leads to raising required disk size
+by 22% in comparison with using shards with 16k ledgers. In the same way,
 using shards with 16k ledgers leads to raising required disk space by 12%
 in comparison with using shards with 16k ledgers. Note that increase ratio 12%
 is much less than 22% so using 32k-ledgers shards will bring us not so much
-economy in disk space. At the same time, we also may take into consideration
-computational power of nodes. For example, my machine with 32G memory is best
-for acquire shards with 16k ledgers, but acquiring shards with 32k ledgers
-is probably problematic for such a machine. If we decide to use 32k ledgers
-per shard, then we should consider 64G minimum memory capacity for a node,
-and economy of disk size will be not so much in comparison with 16k ledgers
-shards.
+economy in disk space.
```

Comment:
```using shards with 16k ledgers leads to raising required disk space by 12% in comparison with using shards with 16k ledgers.```

I believe you meant

``````using shards with 16k ledgers leads to raising required disk space by 12% in comparison with using shards with 32k ledgers.```

Reply:
Done.